### PR TITLE
feat(add): Zemismart ZMR4 (_TZ3000_11pg3ima)

### PR DESCRIPTION
**Pull request by Burki24**

### CODE REVIEW

File: src/devices/zemismart.ts [hunk 1]
```diff
@@ -368,6 +368,43 @@ const definitions: Definition[] = [
         description: 'Smart 20A outlet',
         extend: [identify(), tuya.modernExtend.tuyaOnOff({indicatorMode: true, onOffCountdown: true, childLock: true})],
     },
+    {
+        fingerprint: tuya.fingerprint('TS004F', ['_TZ3000_11pg3ima']),
+        model: 'ZMR4',
+        vendor: 'Zemismart',
+        description: 'Wireless switch with 4 buttons',
```
Comment (Koenkk) [inline src/devices/zemismart.ts:375]:
```suggestion
        description: 'Wireless switch with 4 buttons',
        extend: [battery()],
```

And remove battery from exposes, and configure

File: src/devices/zemismart.ts [hunk 2]
```diff
@@ -368,6 +368,43 @@ const definitions: Definition[] = [
...
+                '4_single',
+                '4_double',
+                '4_hold',
+            ]),
+        ],
+        fromZigbee: [fz.battery, tuya.fz.on_off_action],
+        toZigbee: [],
+        configure: async (device, coordinatorEndpoint) => {
+            const endpoint = device.getEndpoint(1);
+            await endpoint.read('genBasic', [0x0004, 0x000, 0x0001, 0x0005, 0x0007, 0xfffe]);
```
Comment (Koenkk) [inline src/devices/zemismart.ts:397]:
```suggestion
            await tuya.configureMagicPacket(device, coordinatorEndpoint);
```

Isn't this the same?

File: src/devices/zemismart.ts (hunk 1 above)
Comment (Burki24) [reply src/devices/zemismart.ts:375]: Runs well, thanks.

**Comment by Koenkk** [comment]
Doesn't this device work with the definition from https://github.com/Koenkk/zigbee-herdsman-converters/blob/4a81f8cf5ea7e1562b1b337a1c4360ae378ea071/src/devices/tuya.ts#L3077 ?

**Comment by Burki24** [comment]
> Doesn't this device work with the definition from
> 
> https://github.com/Koenkk/zigbee-herdsman-converters/blob/4a81f8cf5ea7e1562b1b337a1c4360ae378ea071/src/devices/tuya.ts#L3077
> 
> ?

No, it isn't compatible. I was testing it before I wrote the update into semismart.ts

**Comment by Koenkk** [comment]
I see, thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `MODEL.png` (update `model` accordingly) 
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by Burki24** [comment]
> I see, thanks!
> 
> Could you also submit a picture for the docs? This can be done by:
> 
> 1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
> 2. Go to the `public/images/devices` directory, _Add file_ -> _Upload files_
> 3. Upload the files and press _Commit changes_
> 4. Press _Contribute_ -> _Open pull request_ -> update title/description -> _Create pull request_
> 
> Make sure that:
> 
> * The filename is `MODEL.png` (update `model` accordingly)
> * The size is 512x512
> * The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

Hi @Koenkk ,
is allready done: https://github.com/Koenkk/zigbee2mqtt.io/pull/2870 

Greetings Burkhard

**Pull request merged**